src/components/DashboardNoTeam/ConfirmTeamCreation/ConfirmTeamCreation.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useMutation, gql } from '@apollo/client';
+import { Box, Text, Heading, Button, HStack, VStack } from '@chakra-ui/react';
+
+const ConfirmTeamCreation = props => {
+  const CREATE_TEAM = gql`
+    mutation createTeam($name: String!, $organization: String!) {
+      createTeam(name: $name, organization: $organization, amountRaised: 0, itemsSold: 0) {
+        id
+      }
+    }
+  `;
+
+  const ADD_USER_TO_TEAM = gql`
+    mutation updateUser($id: ID!, $teamId: String) {
+      updateUser(id: $id, teamId: $teamId) {
+        id
+      }
+    }
+  `;
+
+  const [createTeam] = useMutation(CREATE_TEAM);
+  const [addUser] = useMutation(ADD_USER_TO_TEAM);
+
+  const executeQuery = () => {
+    createTeam({
+      variables: { name: props.teamName, organization: props.teamAffiliation },
+    })
+      .then(data => {
+        let teamId = data.data.createTeam.id;
+        addUser({
+          variables: { id: props.userId, teamId: teamId },
```

Comment:
[ES6 ](https://alligator.io/js/object-property-shorthand-es6/
) allows you to omit the value if it's named the same 

```suggestion
          variables: { id: props.userId, teamId },
```